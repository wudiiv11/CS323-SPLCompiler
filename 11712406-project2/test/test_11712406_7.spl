int main() {
    int age[3];
    int abc[3];
    age[3] = 5;
    if (age > abc) {
        int b = 3;
    } else {
        int c = 5;
    }
    return abc[2] + age[1];
}