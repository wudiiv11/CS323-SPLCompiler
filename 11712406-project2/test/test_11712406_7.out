Error type 7 at Line 5: binary operation on non-number variables